servers/services/src/main/java/org/projectnessie/services/impl/TreeApiImpl.java

```diff
@@ -893,14 +893,20 @@ public <R> R getEntries(
         }
       }
 
-      Predicate<KeyEntry> filterPredicate = filterEntries(filter);
+      Predicate<KeyEntry> paramFilterPredicate = filterEntries(filter);
```

Comment:
note that `filterEntries` allows filtering on `KeyEntryForCel` only, which does not have any real `Content` properties, so the predicate can be applied early and doesnt need to wait for the `KeyEntry` with the `Content`

Reply:
Ah good point. Maybe we should group the attributes that can be filtered on in a separate entity that does not expose `getContent()` (maybe a superinterface is enough).

Reply:
IIRC, CEL does not allow reflection/introspection. It can only access what is declared, and that is in `KeyEntryForCel` (no getter for content).